src/tox/provision.py

```diff
@@ -75,6 +75,14 @@ def provision(state: State) -> int | bool:
 
     def add_tox_requires_min_version(requires: list[Requirement]) -> list[Requirement]:
         min_version: Version = state.conf.core["min_version"]
+        # If own version can be a development one or a pre-release, we need to
+        # to only use its base_version for requirements, or pip will never be
+        # able to find a version that is compatible with the requirement.
+        # See https://github.com/tox-dev/tox/issues/2414
+        if min_version.is_devrelease or min_version.is_prerelease:  # noqa: SC200
+            # So we build a PEP 440 version out of the base ones.
+            # https://peps.python.org/pep-0440/#implicit-pre-release-number
```

Comment:
Same as above for this link. The comment above doesn't explain what's this about.